akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/javadsl/ActorTestKit.scala

```diff
@@ -27,6 +27,9 @@ object ActorTestKit {
    * e.g. threads will include the name.
    * When the test has completed you should terminate the `ActorSystem` and
    * the testkit with [[ActorTestKit#shutdownTestKit]].
+   *
+   * Config loaded from `application-test.conf` if that exists, otherwise
+   * using default configuration (from reference.conf, application.conf not used).
```

Comment:
I sometimes see users get confused and add configuration to their project in a `reference.conf` instead of the `application.conf`. Perhaps we should make it extra clear here that that's not intended?

```suggestion
   * using default configuration (from the reference.conf resources that ship with the Akka libraries. The application.conf of your project is not used in this case).
```

Reply:
I agree